tools/pythonpkg/src/include/duckdb_python/import_cache/modules/pandas_module.hpp

```diff
@@ -31,11 +36,11 @@ struct PandasCoreArraysArrowCacheItem : public PythonImportCacheItem {
 	~PandasCoreArraysArrowCacheItem() override {
 	}
 	virtual void LoadSubtypes(PythonImportCache &cache) override {
-		dtype.LoadModule("pandas.core.arrays.arrow.dtype", cache);
+		dtype.LoadModule("pandas.core.dtypes.dtypes", cache);
```

Comment:
You can actually import ArrowDtype from the `pandas` main namespace (everything in `pandas.core` is considered private, and can "break" at any time)

Reply:
Thanks!

I thought this wasn't done because the arrow api is still experimental, good to know we can use `pandas.ArrowDtype` on both 2.0.0 and 2.1.0 👍